packages/test/test-gc-sweep-tests/src/test/gcSweepTest.spec.ts

```diff
@@ -86,27 +86,33 @@ describeNoCompat("GC Sweep tests", (getTestObjectProvider) => {
     let overrideLogger: IgnoreErrorLogger;
 
     const sleep = async (ms: number) => new Promise((resolve) => setTimeout(resolve, ms));
-    // TODO: run the test with multiple seeds consistently
-    const seed = Math.random();
-    const random: IRandom = makeRandom(seed);
 
-    // Improve this to distribute the randomness among relevant GC times
-    const randomInteger = (min: number, max: number): number => {
-        return random.integer(min, max);
-    };
+    let seed = Math.random();
+    let random: IRandom = makeRandom(seed);
 
     // Time spent to run the test. Currently 10 seconds, for better coverage increase this number.
     const testTime = 10 * 1000;
 
+    beforeEach(async () => {
+        seed = Math.random();
+        random = makeRandom(seed);
+    });
+
     // Currently for GC Sweep testing only, run with npm run test. Should not be running in CI
     // Note: can run with npm run test:build to build and run the test
     // TODO: setup test to run in CI
+    const numberOfTests = 10;
+    for (let i = 0; i < numberOfTests; i++) {
     it(`GC Randomization Test with Seed: ${seed}`, async () => {
```

Comment:
Note: whitespace was purposely not added so the PR was more readable

Reply:
Will you have a follow-on that fixes the indentation?

Reply:
I was thinking I would just add the spaces right before merging